kokoro/scripts/linux/build-docker.sh

```diff
@@ -131,6 +131,7 @@ elif [ $TOOL = "cmake-smoketest" ]; then
   git clone https://github.com/KhronosGroup/SPIRV-Headers.git spirv-headers
   git clone https://github.com/google/re2
   git clone https://github.com/google/effcee
+  git clone https://github.com/abseil/abseil-cpp abseil_cpp
```

Comment:
Should we be pinning these to particular revisions?

Reply:
The smoke test is specifically about checking out the latest version of each submodule, and checking if it works. It gives us a heads up if a change in does not work well with the latest. For example, a correct validator change cause a glslang test to fail. Then we know we need to update the glslang test.

We expect this to work best if we have the latest for everything.

Reply:
Ack